base_automation_team_activity/models/ir_actions_server.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Binhex - Adasat Torres
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, fields, models
+
+
+class IrActionsServer(models.Model):
+    _inherit = "ir.actions.server"
+
+    activity_user_type = fields.Selection(
+        selection_add=[
+            ("team", _("Specific Team")),
```

Comment:
```suggestion
            ("team", "Specific Team"),
```
selection options are translated automatically